components/automate-gateway/api/nodes/manager/manager.proto

```diff
@@ -108,6 +166,25 @@ service NodeManagerService {
 			action: "infra:nodeManagers:delete"
 		};
 	};
+
+	/*
+	List of nodemanagers
+
+	Returns a list of nodemanagers matching the query.
+	Supports filtering, sorting, and pagination.
+	Valid filtering fields: 'manager_type'
+	Valid sorting fields: name, type, status, status_message, date_added
```

Comment:
Should use consistent quoting between filtering and sorting fields (either both with quotes or both without).
Also, the above 4 lines are rendered as a paragraph. Those two "Valid..." sections in particular read better with line breaks, though.